lib/inspec/schema/schema.rb

```diff
@@ -0,0 +1,64 @@
+require "json"
+require_relative "./primitives"
+require_relative "./exec_json"
+require_relative "./exec_json_min"
+require_relative "./profile_json"
```

Comment:
Also here, I don't think we need require_relative as require will work fine for these.

Reply:
Changed all requires to `require "lib/inspec/schema/__file__"`